### DISCUSSION TITLE
`PackerUpdate` stuck at a specific amount of plugins

### DISCUSSION THREAD

**Discussion by hgiesel** [wbthomason/packer.nvim#999] [Q&A] [answered]
I currently have 80 packages installed. When I run `PackerUpdate` or `PackerSync`, it will update 72 plugins (in very rare cases 73)

```
 ⟳ nvim-telescope/telescope.nvim: checking current commit...
 ⟳ wellle/targets.vim: checking current commit...
 ⟳ hrsh7th/cmp-nvim-lsp: checking current commit...
 ⟳ anuvyklack/keymap-amend.nvim: checking current commit...
 ⟳ vim-scripts/repmo.vim: checking current commit...
 ⟳ hrsh7th/cmp-path: checking current commit...
 ✓ lukas-reineke/indent-blankline.nvim: already up to date
 ✓ neovim/nvim-lspconfig: already up to date
 ✓ filipdutescu/renamer.nvim: already up to date
 ✓ ellisonleao/gruvbox.nvim: already up to date
 ✓ svermeulen/vim-subversive: already up to date
```

I don't even know how to go about debugging this. I can disable any 8 plugins, and it updates just fine.

**Comment by EdenEast** [comment, marked as answer]
Just a shot in the dark but would setting `max_jobs` to something small like `4` or `5` fix it?

**Reply by danielo515** [reply]
I'm pretty sure that is the problem, because before setting that, i was getting the exact same problem, only that I had 120 plugins and packer was stucking at 80

**Reply by hgiesel** [reply]
Thanks so much! This has blocked me from altering my setup for quite some time, but now it works perfectly again.

In fact, I can still set the max_jobs to quite a high value, so I used `50`:

```lua
local packer = require 'packer'

packer.init {
    max_jobs = 50,
}

return packer.startup(function(use)
    [...]
end)
```
